sql-plan-management.md

```diff
@@ -189,6 +193,18 @@ explain select * from t1,t2 where t1.id = t2.id;
 >
 > 执行 `DROP GLOBAL BINDING` 会删除当前 tidb-server 实例缓存中的绑定，并将系统表中对应行的状态修改为 'deleted'。该语句不会直接删除系统表中的记录，因为其他 tidb-server 实例需要读取系统表中的 'deleted' 状态来删除其缓存中对应的绑定。对于这些系统表中状态为 'deleted' 的记录，后台线程每隔 100 个 `bind-info-lease`（默认值为 `3s`，合计 `300s`）会触发一次对 `update_time` 在 10 个 `bind-info-lease` 以前的绑定（确保所有 tidb-server 实例已经读取过这个 'deleted' 状态并更新完缓存）的回收清除操作。
 
+### 变更绑定状态
+
+{{< copyable "sql" >}}
+
+```sql
+SET BINDING [ENABLED | DISABLED] FOR BindableStmt;
+```
+
+该语句可以在 GLOBAL 作用域内变更指定的执行计划绑定的状态，作用域不可指定，默认作用域为 GLOBAL。
+
+该语句可以将绑定的状态设置成 `Enabled` 或者 `Disabled` 状态，其中只有 `Disabled` 状态的绑定可以被设置成 `Enabled` 状态，只有 `Enabled` 状态的绑定可以被设置成 `Disabled` 状态。如果当前没有可以改变状态的绑定，则会在日志中输出一条内容为 `The memory usage of all available bindings exceeds the cache's mem quota. As a result, all available bindings cannot be held on the cache. Please increase the system variable 'tidb_mem_quota_binding_cache' and execute 'admin reload bindings' to ensure that all bindings are available normally` 的警告日志进行提示。需要注意的是当绑定的状态被设置成 `Disabled` 状态的时候，该绑定不会被查询语句所使用。
```

Comment:
````suggestion
该语句可以将绑定的状态设置成 `Enabled` 或者 `Disabled` 状态。注意，该语句只能将 `Disabled` 状态的绑定设置成 `Enabled` 状态，不能将 `Enable` 状态的绑定设置成 `Disabled` 状态。如果当前没有可以改变状态的绑定，则会在日志中输出一条内容如下的的警告日志进行提示：

```
The memory usage of all available bindings exceeds the cache's mem quota. As a result, all available bindings cannot be held on the cache. Please increase the system variable 'tidb_mem_quota_binding_cache' and execute 'admin reload bindings' to ensure that all bindings are available normally`
```

需要注意的是当绑定的状态被设置成 `Disabled` 状态的时候，该绑定不会被查询语句所使用。
````